tests/test_testRepo.py

```diff
@@ -126,16 +126,15 @@ def testAddDataIdValue(self):
         with self.assertRaises(ValueError):
             addDataIdValue(self.butler, "detector", 101, nonsenseField="string")
 
-        # No skymap defined; should be created automatically.
-        addDataIdValue(self.butler, "tract", 42)
-        (tract_data_id,) = set(self.butler.registry.queryDataIds(["tract"]).limit(1))
-        self.assertEqual(tract_data_id["tract"], 42)
-        # And then a new one created here, since the now-existing one should
-        # have a random name that isn't 'map'.
-        addDataIdValue(self.butler, "tract", 43, skymap="map")
-        self._checkButlerDimension(
-            {"tract", "skymap"}, "tract=43 AND skymap='map'", [{"skymap": "map", "tract": 43}]
-        )
+        # Keywords imply different instruments
+        with self.assertRaises(RuntimeError):
+            addDataIdValue(self.butler, "exposure", 101, instrument="dummyCam", physical_filter="k2020")
+
+        # No skymap defined
+        with self.assertRaises(RuntimeError):
+            addDataIdValue(self.butler, "tract", 42)
+        with self.assertRaises(RuntimeError):
+            addDataIdValue(self.butler, "tract", 43, skymap="map")
```

Comment:
Maybe this should have its own comment:
```suggestion
        # Didn't create skymap "map" first.
        with self.assertRaises(RuntimeError):
            addDataIdValue(self.butler, "tract", 43, skymap="map")
```